README.md

```diff
@@ -9,6 +9,40 @@ A python package for efficient pickling.
 
 ## Installation
 
+```bash
+pip install slim-trees
+# or
+mamba install slim-trees -c conda-forge
+```
+
+## Usage
+
+Using `slim-trees` does not affect your training pipeline.
+Simply replace calls to `pickle.dumps` with `slim_trees.dump_sklearn_compressed`:
+
+```python
+# example, you can also use other Tree-based models
+from sklearn.tree import DecisionTreeClassifier
```

Comment:
Let's maybe change this to `RandomForestRegressor`, the savings for a single tree are not worth the effort 😅